frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -1,28 +1,89 @@
 import { useMemo } from 'react';
-import { List, ListItem } from '@mui/material';
+import { List, ListItem, styled, Typography } from '@mui/material';
 import { useStrategies } from 'hooks/api/getters/useStrategies/useStrategies';
 import { FeatureStrategyMenuCard } from '../FeatureStrategyMenuCard/FeatureStrategyMenuCard';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import useProject from 'hooks/api/getters/useProject/useProject';
+import { IStrategy } from 'interfaces/strategy';
+import useUiConfig from '../../../../../hooks/api/getters/useUiConfig/useUiConfig';
 
 interface IFeatureStrategyMenuCardsProps {
     projectId: string;
     featureId: string;
     environmentId: string;
 }
 
+const StyledTypography = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.smallBody,
+    padding: theme.spacing(1, 2),
+}));
+
 export const FeatureStrategyMenuCards = ({
     projectId,
     featureId,
     environmentId,
 }: IFeatureStrategyMenuCardsProps) => {
     const { strategies } = useStrategies();
+    const { uiConfig } = useUiConfig();
+
+    const preDefinedStrategies = useMemo(() => {
+        return strategies.filter(
+            strategy => !strategy.deprecated && !strategy.editable
+        );
+    }, [strategies]);
 
-    const availableStrategies = useMemo(() => {
-        return strategies.filter(strategy => !strategy.deprecated);
+    const customStrategies = useMemo(() => {
+        return strategies.filter(
+            strategy => !strategy.deprecated && strategy.editable
+        );
     }, [strategies]);
 
+    const { project } = useProject(projectId);
+
+    const strategy = project.environments.find(
+        env => env.environment === environmentId
+    )?.defaultStrategy;
+
+    const defaultStrategy: IStrategy | undefined = strategy
+        ? {
```

Comment:
is this the only representation of the default strategy in the frontend code? or does it exist in other places and can get out of sync?

Reply:
This is only representation if IStrategy for default. yes